src/MacroExpander.js

```diff
@@ -153,38 +172,38 @@ export default class MacroExpander implements MacroContextInterface {
     expandOnce(): Token | Token[] {
         const topToken = this.popToken();
         const name = topToken.text;
-        if (!this.macros.hasOwnProperty(name)) {
+        const expansion = this._getExpansion(name);
+        if (expansion === null) {
             // Fully expanded
             this.pushToken(topToken);
             return topToken;
         }
-        const {tokens, numArgs} = this._getExpansion(name);
         if (this.maxExpand !== Infinity) {
             this.maxExpand--;
             if (this.maxExpand < 0) {
                 throw new ParseError("Too many expansions: infinite loop or " +
                     "need to increase maxExpand setting");
             }
         }
-        let expansion = tokens;
-        if (numArgs) {
-            const args = this.consumeArgs(numArgs);
+        let tokens = expansion.tokens;
+        if (expansion.numArgs) {
+            const args = this.consumeArgs(expansion.numArgs);
```

Comment:
It looks like this renaming doesn't actually change any of the logic, is that correct?

Reply:
Correct.  There was a need for `expansion` being the thing containing the tokens, so I did the rename.  Sorry it's confusing as a diff.

Reply:
It's fine.  The refactor was pretty isolated.  I just wanted to be sure I wasn't missing something here.